src/CardList/index.js

```diff
@@ -96,14 +116,20 @@ export default class ImageList extends Component {
   renderGrid(items) {
     let { layout, columnCount, editor, _fonts } = this.props
 
-    let { fullWidth } = this.state
-    let width = fullWidth / columnCount - 8
+    const rows = this.getRows(items)
+    const columnWidth = { width: `${100 / columnCount}%` }
 
     return (
       <View onLayout={this.handleLayout} style={styles.gridWrap}>
-        {items.map((itm, i) =>
-          this.renderCell(itm, layout, editor, _fonts, width)
-        )}
+        {rows.map((row, i) => (
+          <View key={i} style={styles.row}>
+            {row.map((itm) => (
+              <View style={columnWidth}>
```

Comment:
should this `<View>` also have a `key` attribute as it's directly within the `.map()`? Looking at the previous code, I see `renderCell()` includes a `key` attribute that probably isn't effective anymore.